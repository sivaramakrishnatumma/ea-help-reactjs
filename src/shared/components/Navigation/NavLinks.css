.navbar-nav {
  width: 100%;
  justify-content: flex-end;
}

@media (min-width: 992px) {
  .nav-link {
    padding-left: 1rem !important;
    padding-right: 1rem !important;
  }
}