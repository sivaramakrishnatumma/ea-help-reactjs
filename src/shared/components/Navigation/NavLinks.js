import React from "react";
import { NavLink } from "react-router-dom";

import "./NavLinks.css";

const NavLinks = props => {
  return (
    <React.Fragment>
      <ul className="navbar-nav">
        <li className="nav-item" onClick={props.collapseMenu}>
          <NavLink className="nav-link" to="/games">
            GAMES
          </NavLink>
        </li>
        <li className="nav-item" onClick={props.collapseMenu}>
          <NavLink className="nav-link" to="/accounts">
            ACCOUNTS
          </NavLink>
        </li>
        <li className="nav-item" onClick={props.collapseMenu}>
          <NavLink className="nav-link" to="/contact-us">
            CONTACT US
          </NavLink>
        </li>
      </ul>
    </React.Fragment>
  );
};

export default NavLinks;
