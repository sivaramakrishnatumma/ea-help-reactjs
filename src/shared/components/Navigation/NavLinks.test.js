import React from "react";
import NavLinks from "./NavLinks";
import { shallow } from "enzyme";

describe("NavLinks component test case", () => {
  it("should test NavLinks component with default props", () => {
    const props = {
      collapseMenu: jest.fn()
    };
    const wrapper = shallow(<NavLinks {...props}></NavLinks>);
    expect(wrapper).toMatchSnapshot();
  });
});
