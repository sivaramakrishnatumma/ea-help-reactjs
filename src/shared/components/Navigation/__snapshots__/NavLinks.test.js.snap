// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavLinks component test case should test NavLinks component with default props 1`] = `
<Fragment>
  <ul
    className="navbar-nav"
  >
    <li
      className="nav-item"
      onClick={[MockFunction]}
    >
      <NavLink
        className="nav-link"
        to="/games"
      >
        GAMES
      </NavLink>
    </li>
    <li
      className="nav-item"
      onClick={[MockFunction]}
    >
      <NavLink
        className="nav-link"
        to="/accounts"
      >
        ACCOUNTS
      </NavLink>
    </li>
    <li
      className="nav-item"
      onClick={[MockFunction]}
    >
      <NavLink
        className="nav-link"
        to="/contact-us"
      >
        CONTACT US
      </NavLink>
    </li>
  </ul>
</Fragment>
`;
